src/pages/Dashboard.tsx

```diff
@@ -48,7 +48,7 @@ const Dashboard = () => {
             name={`${user.name} ${user.family_name}`}
             handleClick={handleClick}
             activeTab={tab}
-            entityType={agreement.entityType!}
+            entityType={agreement.entityType}
```

Comment:
```suggestion
            entityType={agreement.entityType!}
```

This was raising just a warning, we can keep the warning at the moment

Reply:
it is ok to remove the non null assertion, because if an admin is viewing a page, there is no entity type.
see https://github.com/pagopa/cgn-onboarding-portal-frontend/tree/PE-556-public-entitites-fix-2 changes where default casing is added